sqlmesh/core/snapshot/definition.py

```diff
@@ -291,7 +304,8 @@ class SnapshotTableInfo(PydanticModel, SnapshotInfoMixin, frozen=True):
     parents: t.Tuple[SnapshotId, ...]
     previous_versions: t.Tuple[SnapshotDataVersion, ...] = ()
     change_category: t.Optional[SnapshotChangeCategory] = None
-    kind_name: ModelKindName
+    kind_name: t.Optional[ModelKindName] = None
+    node_type_: NodeType = Field(default=NodeType.MODEL, alias="node_type")
```

Comment:
Why do we need an alias if the property method just returns the value as is?

Reply:
Why is this a property of the Snapshot and not the `_Node`?

Reply:
Its a property of SnapshotTableInfo, which doesn't have the node to figure out the type.